index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>surveyForm</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <div id="title-div">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+
+    </div>
+    <div class="container">
+    <form action="" class="survey-form" id="survey-form" method="post">
+        <fieldset>
+            <label for="name" id="name-label">Name <input type="text" name="name" id="name" placeholder="Enter your name" required></label>
+            <label for="email" id="email-label">Email <input type="email" email="email" id="email" placeholder="Enter your email" required></label>
+            <label for="age"id="number-label">Age <small>(optional)</small><input type="number" name="age"  id="number" placeholder="Age" min="16" max="100"></label>
```

Comment:
- [x] You have done a very good job requesting this change, however,  Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

for example, you could say: :arrow_down: 

- [ ] Good job so far, :clap: , however, Please ensure to Kindly move the small tag inside the closing label tag, When structuring HTML, it is essential to maintain proper nesting and adhere to the correct order of tags. The `<small>` tag is used to represent smaller text within other content. To move the `<small>` tag inside the closing `<label>` tag, you need to make sure it is placed within the opening and closing `<label>` tags.